arches/app/templates/views/components/photo-workbench.htm

```diff
@@ -114,7 +114,8 @@ <h4>
                                 <span data-bind="text: $root.translations.addingPhotosOptional"></span>
                             </p>
                             <p>
-                                <span data-bind="text: $root.translations.maxPhotoSizeWarning"></span>
+                                <span data-bind="text: $root.translations.maxPhotoSizeWarning + ' : '"></span>
+                                <span data-bind="text: maxFilesize() + ' MB'"></span>
```

Comment:
Nice catch!

This won't give the translator much flexibility about where to put the size in the sentence, and probably won't work for RTL. I'm wondering if you considered including a placeholder in the string (like `{maxFilesize}`), and then replacing the placeholder in the javascript?

Reply:
Absolutely.. ..makes good sense to me!  

Amended to use placeholder!  ^_^